### PR TITLE
[Network] Bump network to 2021-05-01

### PR DESCRIPTION
**Pull request by 00Kai0**
**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Review by kairu-ms — APPROVED** [review]

**Comment by yonzhan** [comment]
Network

**Review by calvinhzy — APPROVED** [review]

**Review by jiasli — APPROVED** [review]

File: src/azure-cli/requirements.py3.Darwin.txt [hunk 1]
```diff
@@ -61,7 +61,7 @@ azure-mgmt-media==7.0.0
 azure-mgmt-monitor==2.0.0
 azure-mgmt-msi==0.2.0
 azure-mgmt-netapp==5.1.0
-azure-mgmt-network==19.1.0
+azure-mgmt-network==19.2.0
```
Comment (kairu-ms) [inline src/azure-cli/requirements.py3.Darwin.txt:64]:
```suggestion
azure-mgmt-network==19.3.0
```

File: src/azure-cli/requirements.py3.Linux.txt [hunk 2]
```diff
@@ -61,7 +61,7 @@ azure-mgmt-media==7.0.0
 azure-mgmt-monitor==2.0.0
 azure-mgmt-msi==0.2.0
 azure-mgmt-netapp==5.1.0
-azure-mgmt-network==19.1.0
+azure-mgmt-network==19.2.0
```
Comment (kairu-ms) [inline src/azure-cli/requirements.py3.Linux.txt:64]:
```suggestion
azure-mgmt-network==19.3.0
```

File: src/azure-cli/requirements.py3.windows.txt [hunk 3]
```diff
@@ -61,7 +61,7 @@ azure-mgmt-media==7.0.0
 azure-mgmt-monitor==2.0.0
 azure-mgmt-msi==0.2.0
 azure-mgmt-netapp==5.1.0
-azure-mgmt-network==19.1.0
+azure-mgmt-network==19.2.0
```
Comment (kairu-ms) [inline src/azure-cli/requirements.py3.windows.txt:64]:
```suggestion
azure-mgmt-network==19.3.0
```

**Pull request merged**